mips/gt64120.c

```diff
@@ -1,5 +1,6 @@
-/* Heavily inspired by FreeBSD / NetBSD `gt_pci.c` file. */
-
+/* GT64120 PCI bus driver
+ *
+ * Heavily inspired by FreeBSD / NetBSD `gt_pci.c` file. */
```

Comment:
We could mention that this file is a merge of FreeBSDs `/sys/mips/malta/gt.c`, `/sys/mips/malta/gt_pci.c` and it is related in some way to `/sys/dev/pci/*`.